lib/tstracker/tracker.go

```diff
@@ -31,6 +31,14 @@ func (tcs *tcs) CurrentDbTipSetKey(ctx context.Context) (*types.TipSetKey, abi.C
 	return &tsk, epoch, nil
 }
 
+func (tcs *tcs) GetCurrentTipset(ctx context.Context) []cid.Cid {
+	tsk, _, err := tcs.CurrentDbTipSetKey(ctx)
+	if err != nil {
+		return []cid.Cid{}
+	}
+	return tsk.Cids()
+}
+
```

Comment:
Huh... @willscott are you sure you need this? The tsk is what you get by feeding these CID's to the tipset maker no...?

Reply:
that's the function statediff is actually using 😁

Reply:
Ok, I will make a method like this then in the lib, it's pretty silly to do that twice ;)

Reply:
don't worry about it. i can wrap it in my data source thing also. other libraries should be looking for more useful errors, which is what your current interface has. mine is just sloppy about it.